### PR TITLE
Performance and accessibility

### PR DESCRIPTION
**Pull request by victoriabernard92**
Optimizations based on lighthouse and wegpagetest suggestions.

Results after:
Screen Shot 2020-03-03 at 8 36 40 AM
Screen Shot 2020-03-03 at 8 36 32 AM

### CODE REVIEW

File: src/components/Layout.tsx [hunk 1]
```diff
@@ -41,6 +42,7 @@ const Layout: React.FC<LayoutProps> = ({ title, children }) => {
         <script
           type="text/javascript"
           src={withAssetPrefix('/js/docsearch.min.js')}
```
Comment (signalnerve) [inline src/components/Layout.tsx]: it looks like this file was removed in this pr?

File: src/components/Seo.tsx [hunk 2]
```diff
@@ -7,8 +7,14 @@
 
 import React from 'react'
 import Helmet from 'react-helmet'
-import { useStaticQuery, graphql } from 'gatsby'
+import { useStaticQuery, graphql, withAssetPrefix } from 'gatsby'
+// import WebFont from 'webfontloader'
```
Comment (signalnerve) [inline src/components/Seo.tsx]: should this be uncommented?

Comment (signalnerve) [reply src/components/Seo.tsx]: ah, nvm, i see `OpenSans.css` further down in the PR – maybe this should just be removed entirely?

File: src/components/Layout.tsx (hunk 1 above)
Comment (victoriabernard92) [reply src/components/Layout.tsx]: Good eye ! It was just minified not deleted though

File: src/components/Seo.tsx (hunk 2 above)
Comment (victoriabernard92) [reply src/components/Seo.tsx]: Yes done!

File: src/components/DocSearch.tsx [hunk 3]
```diff
@@ -20,12 +20,15 @@ const DocSearch: FunctionComponent<{ initial?: boolean }> = ({
   }, [])
 
   return (
+    <>
```
Comment (signalnerve) [inline src/components/DocSearch.tsx]: looks like this needs re-indentation

File: src/components/Layout.tsx (hunk 1 above)
Comment (signalnerve) [reply src/components/Layout.tsx]: ahhh i see!

File: src/components/Seo.tsx [hunk 4]
```diff
@@ -7,7 +7,7 @@
 
 import React from 'react'
 import Helmet from 'react-helmet'
-import { useStaticQuery, graphql } from 'gatsby'
+import { useStaticQuery, graphql, withAssetPrefix } from 'gatsby'
```
Comment (signalnerve) [inline src/components/Seo.tsx]: looks like we aren't using this but not high priority

**Pull request merged**